client/src/components/kvEditor/KVEditorRow.tsx

```diff
@@ -195,6 +202,16 @@ function KVEditorRow({
     <div key={`${name}-${i}`} className={styles.row}>
       {!readOnly ? (
         <>
+          {canDisableRow && (
+            <input
+              type="checkbox"
```

Comment:
I think the checkbox should be next to the delete icon on the right hand side instead.